packages/react/src/components/modal/modal.tsx

```diff
@@ -81,8 +80,7 @@ interface HeaderProps extends ContentProps {
     isTopScrolled?: boolean;
 }
 const Header = styled.header<HeaderProps>`
-    border-bottom: 1px solid ${({ isTopScrolled, theme }) => (isTopScrolled ? theme.component['modal-header-border-bottom-color'] : 'transparent')};
-    padding: ${getTopPadding} ${getPadding} var(--spacing-2x);
+    border-bottom: 1px solid ${({ isTopScrolled, theme }) => (isTopScrolled ? theme.component['modal-border-color'] : 'transparent')};    padding: ${getTopPadding} ${getPadding} var(--spacing-2x);
```

Comment:
Ramener sur 2 lignes
```suggestion
    border-bottom: 1px solid ${({ isTopScrolled, theme }) => (isTopScrolled ? theme.component['modal-border-color'] : 'transparent')};
    padding: ${getTopPadding} ${getPadding} var(--spacing-2x);
```

Reply:
woops! Je suis surpris que ESlint ou stylelint m'a pas flaggé